domain-impl/src/test/java/com/wednesday/template/domain/lastFm/models/AlbumTestModels.kt

```diff
@@ -0,0 +1,18 @@
+package com.wednesday.template.domain.lastFm.models
+
+import com.wednesday.template.domain.lastFm.Album
+
+val album = Album(
+    mbid = "",
+    name = "",
+    artist = "",
+    url = "",
+    sImageSize = "",
+    sImageText = "",
+    mImageSize = "",
+    mImageText = "",
+    lImageSize = "",
+    lImageText = "",
+    xlImageSize = "",
+    xlImageText = ""
+)
```

Comment:
Insted of empty strings put some dummy data in so that it emulates the real response more closely.